### ISSUE TITLE
`notary verify` not responding.

### ISSUE THREAD

**Issue by waveywaves** [theupdateframework/notary#1541]
I have created a collection, added a target to the collection and am able to list it or look it up successfully with the commands below.
```
$ notary init example.com/collection
$ notary add example.com/collection chart1 wordpress-8.0.1.tgz
$ notary list example.com/collection
$ notary lookup example.com/collection chart1
```

Aprat from that the `verify` subcommand doesn't seem to be responding properly.
```
$ notary verify example.com/collection chart1 
```

Upon trying to debug, the command seems to get stuck and not exit. This seems like a bug to me. 
```
$ notary verify example.com/collection chart1  -D
DEBU[0000] Using the following trust directory: /root/.notary
```

I am guessing this might be because I haven't provided anything to the command to verify the target against but if that is the case the command should exit and I should be getting an error back. From the looks of it, the server is not being called after running this command.

**Comment by chaospuppy** [comment]
Yeah, one year on and the `notary verify` subcommand does nothing.
